src/shared/src/AuditLogDynamoGateway/AwsAuditLogDynamoGateway.ts

```diff
@@ -50,6 +50,19 @@ export default class AwsAuditLogDynamoGateway extends DynamoGateway implements A
     return message
   }
 
+  async updateSanitiseCheck(messageId: string, nextSanitiseCheck: Date): PromiseResult<void> {
+    const options: UnconditionalUpdateOptions = {
+      keyName: this.tableKey,
+      keyValue: messageId,
+      updateExpression: "SET nextSanitiseCheck = :value",
+      updateExpressionValues: { ":value": nextSanitiseCheck.toISOString() }
+    }
```

Comment:
The version number is missing. It might be working perfectly fine now but as the application grows over time and new features are implemented, there might be cases where concurrent updates happen which results in invalid data.

Reply:
Agree. This was because we got frustrated with `DynamoGateway`'s implementation as discussed. [We've found a better way now](https://github.com/ministryofjustice/bichard7-next-audit-logging/pull/295#discussion_r875680253)- this has all been removed 👍